routes/api.php

```diff
@@ -13,20 +16,19 @@
 | routes are loaded by the RouteServiceProvider within a group which
 | is assigned the "api" middleware group. Enjoy building your API!
 |
-*/
+ */
 
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
 
-Route::prefix('v2')->group(function(){
-    Route::get('getToDo',[MainController::class,'getToDo']);
-    Route::post('updateToDo',[MainController::class,'updateToDo']);
-    Route::delete('deleteToDo',[MainController::class,'deleteToDo']);
-    Route::put('newToDo',[MainController::class,'newToDo']);
+Route::prefix(VERSION)->group(function () {
+    Route::prefix('/todos')->group(function () {
+        Route::get(ID, [MainController::class, 'getToDo']);
```

Comment:
route 上的變數一般比較少會直接用常數處理（不是不行，只是好像比較少見）
```suggestion
        Route::get({id}, [MainController::class, 'getToDo']);
```